src/status_im/ui/screens/ens/views.cljs

```diff
@@ -0,0 +1,453 @@
+    label]])
+
+(defn- section [{:keys [title content]}]
+  [react/view {:style {:margin-horizontal 16 :align-items :flex-start}}
+   [react/text {:style {:color colors/gray :font-size 15}}
+    title]
+   [react/view {:margin-top 8 :padding-horizontal 16 :padding-vertical 12 :border-width 1 :border-radius 12 :border-color colors/gray-light}
+    [react/text {:style {:font-size 15}}
+     content]]])
+
+;; Username
+
+(views/defview username []
+  (views/letsubs [{:keys [username address public-key]} [:ens.registration/screen]]
+    [react/view {:style {:flex 1}}
+     [toolbar/simple-toolbar
+      username]
+     [react/scroll-view {:style {:flex 1}}
+      [react/view {:style {:flex 1 :margin-horizontal 16}}
+       [react/view {:flex-direction :row :align-items :center :margin-top 20}
+        [react/view {:style {:margin-right 16}}
+         [components.common/logo
+          {:size      40
+           :icon-size 16}]]
+        [react/text {:style {:typography :title :text-align-vertical :center}}
+         (str (i18n/label :t/ens-10-SNT) ", deposit unlocked")]]]
+      [react/view {:style {:margin-top 22}}
+       [section {:title   (i18n/label :t/ens-wallet-address)
+                 :content (ethereum/normalized-address address)}]
+       [react/view {:style {:margin-top 14}}
+        [section {:title   (i18n/label :t/key)
+                  :content public-key}]]
+       [react/view {:style {:margin-top 16}}
+        [list/big-list-item {:text          (i18n/label :t/ens-remove-username)
+                             :subtext       (i18n/label :t/ens-remove-hints)
+                             :text-color    colors/gray
+                             :action-fn     #(re-frame/dispatch [:navigate-to :ens-register])
+                             :icon          :main-icons/close
+                             :icon-color    colors/gray
+                             :hide-chevron? true}]
+        [list/big-list-item {:text          (i18n/label :t/ens-release-username)
+                             :text-color    colors/gray
+                             :subtext       (i18n/label :t/ens-locked)
+                             :action-fn     #(re-frame/dispatch [:navigate-to :ens-register])
+                             :icon          :main-icons/delete
+                             :icon-color    colors/gray
+                             :active?       false
+                             :hide-chevron? true}]]]]]))
+
+;; Terms
+
+(defn- term-point [content]
+  [react/view {:style {:flex 1 :margin-top 24 :margin-horizontal 16 :flex-direction :row}}
+   [react/view {:style {:width 16 :margin-top 8}}
+    [react/view {:style {:background-color colors/gray :width 4 :height 4 :border-radius 25}}]]
+   [react/text {:style {:flex 1 :font-size 15}}
+    content]])
+
+(defn- etherscan-url [address]
+  (str "https://etherscan.io/address/" address))
+
+(views/defview terms []
+  (views/letsubs [{:keys [contract]} [:get-screen-params :ens-terms]]
```

Comment:
why is this in get-screen-params here?

Reply:
That's how we pass arguments to screens?

Reply:
since @flexsurfer subs PR we stop using any kind of getter subscriptions, just create a simple sub like :ens/terms

Reply:
i believe for screens it works a little bit different, btw if this is a current screen you can use`[:get-screen-params]` and it will return data for current screen